src/python/pants/help/help_printer.py

```diff
@@ -49,6 +51,7 @@ def __init__(
             "subsystems",
             "targets",
             "tools",
+            "json-schema",
```

Comment:
This is not ideal, but we also want to include reserved names here:
https://github.com/pantsbuild/pants/blob/ceaa203f94762477ac05b9a1fc747d60e5ddc29d/src/python/pants/build_graph/build_configuration.py#L31-L33

Reply:
Oh perfect, forgot about this one, done!